connector_woocommerce/components/mapper.py

```diff
@@ -0,0 +1,30 @@
+# © 2009 Tech-Receptives Solutions Pvt. Ltd.
+# © 2018 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.component.core import AbstractComponent
+
+
+class WooImportMapper(AbstractComponent):
+    _name = 'woo.import.mapper'
+    _inherit = ['base.woo.connector', 'base.import.mapper']
+    _usage = 'import.mapper'
+
+
+class WooExportMapper(AbstractComponent):
+    _name = 'woo.export.mapper'
+    _inherit = ['base.woo.connector', 'base.export.mapper']
+    _usage = 'export.mapper'
+
+
+def normalize_datetime(field):
```

Comment:
this method is useless, it was for Magento which sometimes returns a date/time full of zeroes.